### DISCUSSION TITLE
Product Research Questions

### DISCUSSION THREAD

**Discussion by wswitzer** [mozilla/hubs#5673] [Hubs Cloud Questions] [answered]
Hey all,

I'm researching Hubs to see if it's going to fit what we're looking for. And this was the only place there seemed to be people to ask about it. If there's a better place, please point me in that direction. 

We're a spiritual organization with an online community. I don't have the stats for total users, but it's in the multiple hundreds, with probably a hundred regular users.

Hubs seems like it would be a really cool way for everyone to join online for us.

Here are some questions I have coming up that would be very helpful to have the answers to in order to move forward.

Can Hubs somehow stream out live video meetings to 3rd party clients. Or can we stream into Hubs with things such as Zoom or MS Teams? And if so, are there controls to participate in the meeting or just view? If the functionality isn't there, is this something that we could add for functionality in a custom client?

Can Hubs Cloud run on Azure or another provider or only AWS/Digital Ocean?

I have thoughts of integrating user accounts with those in our existing community? Are these exposed and accessible somehow? API?

I think that's it for now. Thank you for any support you may be able to give.
Wesley

**Comment by pattersonbl2** [comment, marked as answer]
We currently don't have the integration with 3rd party streaming platform. The closest thing to this would be sharing your screen with zoom running on your local machine. 

We only have an AWS marketplace offering currently,  We plan to be more cloud agnostic in the future. We currently don't have a timeline for this.  

You may want to join our community discord. https://discord.com/invite/dFJncWwHun   We don't have these API but a lot of devs in the community are building great things that could help.

**Reply by wswitzer** [reply]
Thank you. One other question. Is it possible to create our own landing pages and to direct users to specific rooms with a default avatar? Meaning they don't have to go through any sort of setup process.
